src/main/java/it/pagopa/transactions/exceptions/ProcessingErrorException.java

```diff
@@ -0,0 +1,9 @@
+package it.pagopa.transactions.exceptions;
+
+public class ProcessingErrorException extends RuntimeException {
```

Comment:
This exception is mapped to 422 HTTP response code by code [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/0bce94e59031680a7dec2d6ed9ed80928483bebf/src/main/java/it/pagopa/transactions/controllers/v1/TransactionsController.java#L501-L522). Is this wanted or is a side-effect?

Reply:
It is wanted. 
In this exception I basically want to track this edge case (send payment result for a transction without an authorization, really not possibile). Is this a little bit different from the Already processed exception: here we map the fact that we don't expect this call from nodo simply because the transaction isn't yet authorized. So the 422 response code is ok.